### PR TITLE
track successful reflector uploads in sqlite

### PR DESCRIPTION
**Pull request by jackrobison**
-add `reflected_stream` table to the database to track reflector uploads so we don't check reflector server unnecessarily 
-increase the default `auto_re_reflect_interval` to a day

### CODE REVIEW

File: lbrynet/conf.py [hunk 1]
```diff
@@ -283,7 +283,7 @@ def _convert_value(value):
     # if reflect_uploads is True, send files to reflector (after publishing as well as a
     # periodic check in the event the initial upload failed or was disconnected part way through
```
Comment (shyba) [inline lbrynet/conf.py]: maybe update this comment? `reflect_uploads` doesn't control the re-reflect anymore

Comment (shyba) [reply lbrynet/conf.py]: Sorry, it also does, but this became a bit confusing. Maybe the `# set to 0 to disable` added below is enough. Or explain that they both need to be active for enabling re-reflect.

**Comment by shyba** [comment]
Except for that detail on the older comment, the code looks very good to me. The old comment is not even wrong, just confusing.

File: lbrynet/conf.py (hunk 1 above)
Comment (jackrobison) [reply lbrynet/conf.py]: If `reflect_uploads` is true, streams made with `publish` will be uploaded to reflector. If `reflect_uploads` is true and `auto_re_reflect_interval` > 0 files in the file manager will be reflected.

**Pull request merged**